### ISSUE TITLE
Console Logging in format Function

### ISSUE THREAD

**Issue by KevinKhi** [martpie/react-slugify#17]
### Description
After upgrading from `react-slugify` version 3.0.2 to 4.0.0, I noticed that the library logs the harmonized string to the console.

```ts
function format(
  input: string,
  delimiter: string,
  ignoreInvalid = false
): string {
  const harmonized = stripAccents(input).trim().toLowerCase();
  console.log(harmonized); // <-- 
  const safeDelimiter = getSafeRegexpString(delimiter);
  ...
};
```

**Comment by martpie** [comment]
oops

**Comment by martpie** [comment]
Fixed in `4.0.1`, sorry for the inconvenience!
